apps/src/netsim/NetSimLogPanel.js

```diff
@@ -53,6 +54,20 @@ var MESSAGE_SLIDE_IN_DURATION_MS = 400;
  */
 
 /**
+ * @function
+ * @name INetSimLogPanel#getHeight
+ * @returns {number} vertical space that panel currently consumes (including
+ *          margins) in pixels.
+ */
+
```

Comment:
Nope!  These comments are part of [defining an interface](http://usejsdoc.org/tags-interface.html) [`INetSimLogPanel`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimLogPanel.js#L30) in JSDoc that gets impelmented by both [`NetSimLogPanel`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimLogPanel.js#L65) and [`NetSimBitLogPanel`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/NetSimBitLogPanel.js#L32).  It's not currently enforced by our compiler in any way but it seemed responsible to document somewhere that these things are partially interchangeable.